scripts/deploy-kubefed.sh

```diff
@@ -173,6 +177,13 @@ if [[ ! "${USE_LATEST}" ]]; then
   cd -
   ${DOCKER_PUSH_CMD}
 fi
+
+# Use KIND_LOAD_IMAGE=y DOCKER_PUSH= ./scripts/deploy-kubefed.sh <image> to load
```

Comment:
This is important enough to be mentioned in docs/environments/kind.md or docs/development.md somewhere, no? The documentation is already pretty convoluted...

Reply:
Updated development doc.